### PR TITLE
add open source gpw required intents to minimal owl

### PR DESCRIPTION
**Pull request by sfeng88**
- [x] joeferner
- [x] mwizeman joeybrk372 jharwig sfeng88

Testing Instructions:
- Following the [getting started](http://docs.visallo.org/getting-started.html) instructions, make sure that all of the open source gpws requiring intents work (email extractor, phone number extractor, and zip code extractor)

Points of Regression: N/A

CHANGELOG
Added: Missing required intents into minimal owl file for GPWs

### CODE REVIEW

File: config/ontology-minimal/minimal.owl [hunk 1]
```diff
@@ -27,7 +27,6 @@
     <!-- http://visallo.com/minimal#rawContainsImageOfEntity -->
 
     <owl:ObjectProperty rdf:about="http://visallo.com/minimal#rawContainsImageOfEntity">
-        <rdfs:domain rdf:resource="http://visallo.com/desktop/example#raw"/>
```
Comment (mwizeman) [inline config/ontology-minimal/minimal.owl]: I don't think it's valid to have a relationship without both a domain and range. I had this situation (due to a typo) with a system that I upgraded to 4.0 and it caused errors in the UI. I think we should probably update `OntologyRepositoryBase` to not allow this situation.

File: config/ontology-minimal/minimal.owl [hunk 2]
```diff
@@ -218,24 +217,38 @@
...
+        <rdfs:label xml:lang="en">Phone Number</rdfs:label>
+    </owl:Class>
 
+    <!-- http://visallo.org/minimal#zipcode -->
 
-    <!-- http://visallo.com/minimal#location -->
+    <owl:Class rdf:about="http://visallo.org/minimal#zipcode">
+        <visallo:color>rgb(160, 138, 206)</visallo:color>
+        <visallo:intent>zipCode</visallo:intent>y
+        <rdfs:label xml:lang="en">Zipcode</rdfs:label>
```
Comment (mwizeman) [inline config/ontology-minimal/minimal.owl]: Should be `Zip Code`

File: config/ontology-minimal/minimal.owl (hunk 1 above)
Comment (sfeng88) [reply config/ontology-minimal/minimal.owl]: done.

**Comment by sfeng88** [comment]
@mwizeman done making the changes we discussed

**Pull request merged**